umbraco/Umbraco-CMS
# Management api should use datetimeoffset

### Description
Builds on top of https://github.com/umbraco/Umbraco-CMS/pull/16170

This updates datetime to datetimeoffset in the management api and uses the LocalDateTime when using it.

### Test
- Ensure the endpoints continue to work
- Try to use times with timezone info in management api, and ensure the times are "converted" to the server time
